src/js/actions/MesosActions.js

```diff
@@ -0,0 +1,48 @@
+var AppDispatcher = require("../AppDispatcher");
+var JSONPUtil = require("../helpers/JSONPUtil");
+var MesosEvents = require("../events/MesosEvents");
+
+var MesosActions = {
+  requestState: function (id, url) {
+    JSONPUtil.request(`${url}/state`).then(
```

Comment:
OK ```/state/``` doesn't work with Mesos 0.24, it must be ```/state.json```. Then it works!
Could you test if state.json is also working with 0.25? Then everything is perfect.